src/sensor/utils.py

```diff
@@ -61,4 +66,35 @@ def get_timestamp_from_datetime(dt):
 
 
 def parse_datetime_str(d):
-    return datetime.strptime(d, DATETIME_FORMAT)
+    return datetime.strptime(d, settings.DATETIME_FORMAT)
+
+
+def copy_driver_files():
+    """Copy driver files where they need to go"""
+    for root, dirs, files in os.walk(settings.DRIVERS_DIR):
+        for filename in files:
+            name_without_ext, ext = os.path.splitext(filename)
+            if ext.lower() == ".json":
+                json_data = {}
+                file_path = os.path.join(root, filename)
+                with open(file_path) as json_file:
+                    json_data = json.load(json_file)
+                if type(json_data) == dict and "scos_files" in json_data:
+                    scos_files = json_data["scos_files"]
+                    for scos_file in scos_files:
+                        source_path = os.path.join(
+                            settings.DRIVERS_DIR, scos_file["source_path"]
+                        )
+                        if not os.path.isfile(source_path):
+                            logger.error(f"Unable to find file at {source_path}")
+                            continue
+                        dest_path = scos_file["dest_path"]
+                        dest_dir = os.path.dirname(dest_path)
+                        try:
+                            if not os.path.isdir(dest_dir):
+                                os.makedirs(dest_dir)
```

Comment:
Will this fail for nested directories?

Reply:
`makedirs` will make parent directories: https://docs.python.org/3/library/os.html#os.makedirs